server/.gitignore

```diff
@@ -0,0 +1,83 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
You are already covering a lot of this and of the distro/packaging entries in the root level gitignore through pyo, pyc ignore

Reply:
I think you're confused with the client one.